_episodes/06-includes.md

```diff
@@ -74,14 +74,39 @@ the sites linked in the previous exercise aren't built with Jekyll.
 But the principles behind reusing content apply regardless of
 the particular framework being used to build the site.
 
+## Carrying the Banner
+
 Let's look at an example of how we can create a block of common content
 and reuse it in multiple pages on our site.
 At the moment our pages are quite plain:
 why don't we try adding a banner to the top of each page?
-Begin by adding this HTML to the top of `index.md`:
+
+We're going to add a pre-made banner image to our repository,
+and it is good practice to store all image files in a common folder.
+1. click "Create new file" under the "Add file"
+dropdown on your repository homepage,
+2. in the "Name your file..." box, type `images/`.
+   The folder name should be automatically inserted in the path displayed
+   next to this box for naming the file.
+3. You can then leave the file blank and name it `.gitkeep`.
+   When you commit the changes, the `images` folder will have been
+   added to your repository.
```

Comment:
This seems like a good way to do this but we could also have them drag-drop the image and then rename it into the images folder.  I'm not sure which way would be easier.  Thoughts?

Reply:
This does teach `.gitkeep` which could be good.

Reply:
Let's leave it for now and change if people find this too confusing in the first trial run next week